recipe/meta.yaml

```diff
@@ -55,4 +55,5 @@ about:
 
 extra:
   recipe-maintainers:
+    - arjanz
```

Comment:
@arjanz , could you please let us know if you want to be a maintainer here?

Reply:
Hi @jakirkham, yes that seems like a logical choice as I'm the maintainer of the [py-ed25519-zebra-bindings](https://github.com/polkascan/py-ed25519-zebra-bindings) repos

Reply:
Thanks. @jakirkham the linter did not request Arjan's comment and I just committed, I should have realized that if it is required for staged recipe, it is required for feedstocks, I'll keep this in mind prior to merging next time

Reply:
Thank you both! 🙏

Yes it is best for maintainer updates to include confirmation from those affected by those changes

Reply:
Also, reading the doc more thoroughly, I would have seen that there is actually a process to follow for this purpose. Thank you